tests/resources/decorated_function_samples.py

```diff
@@ -102,3 +102,45 @@ def spark_func(param):
 def standard_py_func(param):
     assert type(param) is dict
     return {'age': param['age']}
+
+
+nested_sample_input = StandardPythonParameterType(
+    {'input1': PandasParameterType(pandas_sample_input),
+     'input2': NumpyParameterType(numpy_sample_input),
+     'input3': StandardPythonParameterType(standard_sample_input),
+     'input0': 0}
+)
+nested_input_data = {'input1': pandas_input_data,
+                     'input2': numpy_input_data,
+                     'input3': standard_sample_input,
+                     'input0': 0}
+nested_sample_output = StandardPythonParameterType(
+    {'output1': PandasParameterType(pandas_sample_output),
+     'output2': NumpyParameterType(numpy_sample_output),
+     'output3': StandardPythonParameterType(standard_sample_output),
+     'output0': 0}
+)
+nested_output_data = {'output1': pandas_output_data,
+                      'output2': numpy_output_data,
+                      'output3': standard_sample_output,
+                      'output0': 0}
+
+
+@input_schema('param', nested_sample_input)
+@output_schema(nested_sample_output)
+def nested_func(param):
+    """
+
+    :param param:
+    :type param: pd.DataFrame
+    :return:
+    :rtype: pd.DataFrame
+    """
+    assert type(param) is dict
+    assert 'input1' in param.keys() and type(param['input1']) is pd.DataFrame
+    assert 'input2' in param.keys() and type(param['input2']) is np.ndarray
+    assert 'input3' in param.keys() and type(param['input3']) is dict
+    output1 = pd.DataFrame(param['input1']['age'])
+    output2 = param['input2']['grades']
+    output3 = {'age': param['input3']['age']}
+    return {'output1': output1, 'output2': output2, 'output3': output3}
```

Comment:
It looks like you're missing 'output0'?

Reply:
and 'intput0'

Reply:
After discussion we decide to ignore those items that are not wrapped by any 'ParameterType'. This is for the sake of backward compatibility, for example you will get the same schema as before if none of the items are wrapped.

Reply:
We shouldn't ignore anything, we just don't attempt to dig deeper, otherwise we're effectively dropping properties which is bad. What you have as `nested_input_data` and `nested_output_data` looks correct, but the actual function handling doesn't match that.

Reply:
The current `standard_sample_input` is an example of that. None of the information in there is wrapped, so we don't dig any into it and the only swagger information we get is that it's an object, but we definitely don't want to be dropping properties: https://github.com/Azure/InferenceSchema/blob/master/tests/resources/sample_standardpy_input_schema.json

Reply:
You are right, actually we are not dropping these unwrapped parameters, we just skip type conversion for them since they will not be described in swagger